### PR TITLE
[SYCL][HIP] Add the query of BF16 support in the HIP plugin

### PR DESCRIPTION
**Pull request by jinz2014**
The PR tries to add the query of BF16 support in the HIP plugin. Thanks for your review.

### CODE REVIEW

File: sycl/plugins/cuda/pi_cuda.cpp [hunk 1]
```diff
@@ -2985,7 +2985,7 @@ pi_result cuda_piEnqueueKernelLaunch(
             return PI_ERROR_INVALID_WORK_GROUP_SIZE;
 
           if (local_work_size[dim] > maxThreadsPerBlock[dim])
-            return PI_ERROR_INVALID_WORK_ITEM_SIZE;
+            return PI_ERROR_INVALID_WORK_GROUP_SIZE;
```
Comment (jchlanda) [inline sycl/plugins/cuda/pi_cuda.cpp:2988]: This is part of https://github.com/intel/llvm/pull/8483, could you please update this PR.

Comment (jinz2014) [reply sycl/plugins/cuda/pi_cuda.cpp]: Sorry, I tried to remove the content of *8483 from the current PR.

**Comment by npmiller** [comment]
The query looks good but are the bf16 math functions already working with HIP? Don't we need some compiler and/or libclc support for this? The bf16 tests in `intel/llvm-test-suite` are currently marked unsupported for HIP afaik.

**Comment by jinz2014** [comment]
This PR will be blocked until BF16 is supported in the SYCL compiler.

**Comment by jchlanda** [comment]
> This PR will be blocked until BF16 is supported in the SYCL compiler.

You might want to use `Convert to draft` link, to mark that this PR is not yet ready to be reviewed/merged.

**Pull request closed without merging**